README.md

```diff
@@ -1,19 +1,79 @@
 
+# GitHub Workflow
 
+---
+### GitHub UI Instructions
+---
 
-# GitHub Workflow
+1. Fork this repository<br>
+   *Done from NSC GitHub Repository (click link below)*  
+   <a href="https://github.com/NSC-Computer-Science-Club/ForkPracticeCareerPrep" target="_blank">https://github.com/NSC-Computer-Science-Club/ForkPracticeCareerPrep</a>
+   ![Highlighting Fork Button Location](readme_images/fork.png)
+
+
+2. The repo will appear under your profile. Click this repository (We will do our work from here).
+
+   ![](readme_images/myProfileRepo.png)
+
+
+   > *For a Deeper dive Using Git Bash command line. Follow Steps **A** thru **G** Under **Terminal Commands*** below
+
+3. Click the green "Add file" button. Select "Create new file"
+
+   ![](readme_images/Adding%20file%20from%20UI.png)
+
+4. At the top of the file will be a textbox to name your file. Include the subfolder. Typing the name of the subfolder and then a forward-slash will indicate a new folder. Then type readme.md
+
+   ![](readme_images/newFile.png)
+   ![](readme_images/newFileWithFolder.png)
+
+5. Then add markdown of your choosing to illustrate what you learned. Here is a <a href="https://stackedit.io" target="_blank">web tool</a> to help with markdown.
+
+6. Once you are satisfied with your changes. Save your file by scrolling down and clicking the green "Commit new file" button.
+
+   ![](readme_images/commit.png)
+
+7. From your GitHub Profile submit a pull request
+
+   ![](readme_images/pullRequest.png)
+
+---
+### Terminal Commands
+---
+**Switch to the Git Bash terminal**
+
+   On your computer navigate to the folder where you want to store your copy within the Git Bash terminal. Cloning will create a subfolder including all the files within the repository.
+
+   A. Clone this repository `git clone https://github.com/NSC-Computer-Science-Club/ForkPracticeCareerPrep.git`
+
+   ![Highlighting Cloning option in GitHub](readme_images/clone.png)
+
+**Switch to your file management system**
+
+
+   B. Create a subfolder within this copied repository. Name it your GitHub profile user name.
+
+   C. In this folder create a README.md file
+
+   D. Open this file and add your markdown and save the file.
+
+
+**Switch to the GitBash terminal**
+
+
+   E. add your changes from the terminal `git add .`
+
+   F. Commit your changes `git commit -m "My created subfolder with markdown file."`
+
+   G. Push your changes to your GitHub Profile repository `git push origin main`
 
-- [ ] Fork this repository
-- [ ] Clone this repository
-- [ ] Create subfolder named with your GitHub id
-- [ ] In this folder create a README.md file
-- [ ] Commit your changes
-- [ ] Push to the repository on your GitHub Profile
-- [ ] Go to you GitHub Profile submit a pull request
+---
+> Continue with **step 7** above
+---
 
-Happy to have yall here <3
+Happy to have yall here :heart:
 
-Good luck :)
+Good luck :smile:
 
 You got this
 
```

Comment:
Maybe there could be an additional section at the bottom for reference links, like for Markdown? This is a good one for commonly used Markdown syntax for READMEs https://docs.github.com/en/get-started/writing-on-github/getting-started-with-writing-and-formatting-on-github/basic-writing-and-formatting-syntax

Reply:
Some other potential links to go into a reference section:

Link to Beck’s Git Intro Video: [Working Together with Github - 30 Minute Virtual TechBytes](https://www.youtube.com/watch?v=UDlIJkvOOPw&list=PL6k4gMRDdFjhHvq9Oulmyph2LezQMU4lg&index=2&t=2s)
Link to Fork Me Repo: https://github.com/atapas/fork-me
Link to GitHub Profile Generator: https://rahuldkjain.github.io/gh-profile-readme-generator/
Link to Good First Issue: https://goodfirstissues.com/

Reply:
@theGaryLarson I also had an idea that if we're talking about collaboration, Open Source, and the Good First Issue tag, we should have an example of the Good First Issue tag. I created a sample issue with the Good First Issue tag (you can find in the Issues tab).